aikiwaw/urls.py

```diff
@@ -18,6 +18,8 @@
         url(r'^training/(?P<slug>[\w\-_]+)/$', TrainingDetail.as_view(), name='training-detail'),
         url(r'^trainings/?$', TrainingList.as_view(), name='training-list'),
         url(r'^trainings/(?P<page>[0-9]+)/$', TrainingList.as_view(), name='training-list'),
+        url(r'^technique/(?P<slug>[\w\-_]+)/(?P<page>[0-9]+)/$', TechniqueList.as_view(), name='technique-list'),
+        url(r'^technique/(?P<slug>[\w\-_]+)/$', TechniqueList.as_view(), name='technique-list'),
```

Comment:
A o to Ci chodzi. Ale mimo wszystko jest to brzydkie rozwiazanie i na pewno da się inaczej.